README.CentOS7.md

```diff
@@ -78,7 +78,7 @@ Then follow instructions to create a password for the root user.
 
 ## NodeJS
 ```bash
-sudo yum install nodejs
+curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -
```

Comment:
Running this line does not finish the installation of nodejs, such as output:
"## Run `sudo yum install -y nodejs` to install Node.js 8.x LTS Carbon and npm."
To fix, after this line, please add the following:
`sudo yum install -y nodejs`

Reply:
@coreyAbout updated.

Reply:
No, this is just reversing the previous change and will install nodejs version that come with CentOS.
If you don't believe me, look at the script from previous line and also https://github.com/nodesource/distributions/blob/master/README.md

Reply:
@PapillonMcGill oops, fixed.